packages/elements/src/calendar/index.ts

```diff
@@ -1377,6 +1400,7 @@ export class Calendar extends ControlElement implements MultiValue {
     const isSelectable = isSelection && !cell.disabled;
     const isSelected = cell.selected ? 'true' : 'false';
     const isActive = cell.active ? 0 : -1;
+    const cellContent = cell.value ? html`<slot name=${cell.value}>${cell.text}</slot>` : cell.text;
```

Comment:
what this short if mean?

Reply:
Without `fillCells` property, head and tail cell will not contain `cell.value`. So, it better to not leave `slot` with empty value in the template.